src/app/base/components/AppSidePanel/AppSidePanel.tsx

```diff
@@ -43,11 +48,51 @@ const useCloseSidePanelOnEscPressed = (): void => {
   useOnEscapePressed(() => setSidePanelContent(null));
 };
 
+export const getSidepanelIcon = (sidePanelContent: SidePanelContent | null) => {
+  if (sidePanelContent) {
+    const [, name] = sidePanelContent.view;
+    switch (name) {
+      case SidePanelViews.POWER_OFF_MACHINE_SOFT[1]:
+        return (
+          <TooltipButton
+            iconName="information"
+            message={
+              <>
+                A soft power off generally asks the OS to
+                <br />
+                shutdown the system gracefully before powering off. <br />
+                It is only supported by IPMI
+              </>
+            }
+          />
+        );
+      case SidePanelViews.POWER_OFF_MACHINE[1]:
+        return (
+          <TooltipButton
+            iconName="information"
+            message={
+              <>
+                Power off will perform a hard power off, which occurs
+                immediately <br />
+                without any warning to the OS.
+              </>
+            }
+          />
+        );
+      default:
+        return <></>;
+    }
+  } else {
+    return <></>;
+  }
+};
+
 const AppSidePanelContent = ({
   title,
   size,
   content,
 }: AppSidePanelProps): JSX.Element => {
+  const { sidePanelContent } = useSidePanel();
```

Comment:
This is a presentational component, and state mapping should happen within the container (`AppSidePanel`).